src/main/java/com/logsentinel/client/ApiVerificationControllerApi.java

```diff
@@ -0,0 +1,829 @@
+        if (applicationId != null)
+        localVarHeaderParams.put("Application-Id", apiClient.parameterToString(applicationId));
+
+        Map<String, Object> localVarFormParams = new HashMap<String, Object>();
+
+        final String[] localVarAccepts = {
+            "application/xml", "application/json"
+        };
+        final String localVarAccept = apiClient.selectHeaderAccept(localVarAccepts);
+        if (localVarAccept != null) localVarHeaderParams.put("Accept", localVarAccept);
+
+        final String[] localVarContentTypes = {
+            "application/json", "*/*"
+        };
+        final String localVarContentType = apiClient.selectHeaderContentType(localVarContentTypes);
+        localVarHeaderParams.put("Content-Type", localVarContentType);
+
+        if(progressListener != null) {
+            apiClient.getHttpClient().networkInterceptors().add(new com.squareup.okhttp.Interceptor() {
+                @Override
+                public com.squareup.okhttp.Response intercept(Chain chain) throws IOException {
+                    com.squareup.okhttp.Response originalResponse = chain.proceed(chain.request());
+                    return originalResponse.newBuilder()
+                    .body(new ProgressResponseBody(originalResponse.body(), progressListener))
+                    .build();
+                }
+            });
+        }
+
+        String[] localVarAuthNames = new String[] { "basicAuth" };
+        return apiClient.buildCall(localVarPath, "GET", localVarQueryParams, localVarCollectionQueryParams, localVarHeaderParams, localVarFormParams, localVarAuthNames, progressRequestListener);
+    }
+
+    @SuppressWarnings("rawtypes")
+    private com.squareup.okhttp.Call getResolvedVerificationFailuresUsingGETValidateBeforeCall(String applicationId, final ProgressResponseBody.ProgressListener progressListener, final ProgressRequestBody.ProgressRequestListener progressRequestListener) throws ApiException {
+        
+        // verify the required parameter 'applicationId' is set
+        if (applicationId == null) {
+            throw new ApiException("Missing the required parameter 'applicationId' when calling getResolvedVerificationFailuresUsingGET(Async)");
+        }
+        
+
+        com.squareup.okhttp.Call call = getResolvedVerificationFailuresUsingGETCall(applicationId, progressListener, progressRequestListener);
+        return call;
+
+    }
+
+    /**
+     * Get all resolved verification failures
+     * 
+     * @param applicationId Application ID, identifying a target application (obtained from the API credentials page) (required)
+     * @return CollectionOfResolvedVerificationFailureResponse
+     * @throws ApiException If fail to call the API, e.g. server error or cannot deserialize the response body
+     */
+    public CollectionOfResolvedVerificationFailureResponse getResolvedVerificationFailuresUsingGET(String applicationId) throws ApiException {
```

Comment:
this return type is not okay. I don't know why it was generated like that, but it should be a List